<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewResolutionPropertyWidget</class>
 <widget class="QWidget" name="ViewResolutionPropertyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>27</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1,0,0,0,0">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="pqLineEdit" name="width">
     <property name="placeholderText">
      <string>width</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="pqLineEdit" name="height">
     <property name="placeholderText">
      <string>height</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="scaleTwice">
     <property name="toolTip">
      <string>Scale by 2</string>
     </property>
     <property name="text">
      <string>x2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="scaleHalf">
     <property name="toolTip">
      <string>Scale by 1/2</string>
     </property>
     <property name="text">
      <string>/2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="lockAspectRatio">
     <property name="toolTip">
      <string>Lock aspect ratio</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../Components/Resources/pqComponents.qrc">
       <normaloff>:/pqWidgets/Icons/pqLock24.png</normaloff>:/pqWidgets/Icons/pqLock24.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="pqHighlightablePushButton" name="reset">
     <property name="toolTip">
      <string>Reset to default based on current values</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqLineEdit</class>
   <extends>QLineEdit</extends>
   <header>pqLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>pqHighlightablePushButton</class>
   <extends>QPushButton</extends>
   <header>pqHighlightablePushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Components/Resources/pqComponents.qrc"/>
 </resources>
 <connections/>
</ui>
